### ISSUE TITLE
Plugin Details: Label color is hard to read

### ISSUE THREAD

**Issue by cpapazoglou** [Automattic/wp-calypso#58279]
### Details

The gray labels are pretty hard to read given the font size and color. They’re [even failing WCAG AA](https://href.li/?https://webaim.org/resources/contrastchecker/?fcolor=787C82&bcolor=FFFFFF). Could they be darkened up a bit to make that easier?

![image](https://user-images.githubusercontent.com/12430020/142624886-d2ee72df-c89c-43ce-9174-066cfe933971.png)

### Checklist

- [ ] (my proposal) Let's replace all instances of `var(--studio-gray-40)` and `var(--studio-gray-50)` with `var(--studio-gray-60)`

cc: @SaxonF 

### Related

pdh6GB-6h-p2#comment-315

**Comment by cpapazoglou** [comment]
@SaxonF the ping was just for making sure any future designs are accessibility friendly.

**Comment by SaxonF** [comment]
Sorry I missed this @cpapazoglou ! Great pickup. Any values over 50 should be accessible on white background
